packages/react/src/hooks/use-slash-id.ts

```diff
@@ -13,7 +13,7 @@ export function useSlashID(): UseSlashID {
     [contextValue.sdkState]
   );
   const isAuthenticated = useMemo(
-    () => contextValue.user !== undefined,
+    () => contextValue.user !== undefined && !contextValue.user.anonymous,
```

Comment:
This is a bit of a controversial change IMO

I decided this made sense because `isAuthenticated` previously meant "user instanceof User", but before this change the meaning was `user instanceof User || user instanceof AnonymousUser`

It took me a while to decide on this because `isAuthenticated` is kind of a vague variable name, technically speaking anon users are authenticated and can do things in the API, but they're not authenticated in terms of customer application logic.